packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -1165,7 +1165,7 @@ export class GarbageCollector implements IGarbageCollector {
 		sweepPhaseStats.lifetimeDataStoreCount += sweepPhaseStats.deletedDataStoreCount;
 		sweepPhaseStats.lifetimeAttachmentBlobCount += sweepPhaseStats.deletedAttachmentBlobCount;
 
-		if (this.configs.sweepEnabled) {
+		if (this.configs.shouldRunSweep) {
```

Comment:
This change look correct, right?

Reply:
Yes, this is correct. I missed it in my PR.